domains/javedpension.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "nx0ly"
+    },
+    "record": {
+        "CNAME": "https://nx0ly.github.io/portfolio.github.io/"
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, or anything combination of those. See docs for more information: https://docs.is-a.dev/ Reopen PR when this is fixed.